recipes/vesin/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "vesin" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: python-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bb325eb9f41e6632600379b1babd26264859d68b2bd1a8bff22ef23efbd8a689
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [linux]
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3.9
```

Reply:
Should this be `python >=3.9` or `python >={{ python_min }}`?

Reply:
Sounds like the linter does not like either of these and fails with `Non noarch packages should have python requirement without any version constraints.`